packages/perseus/src/widgets/radio/__stories__/option-status.stories.jsx

```diff
@@ -3,132 +3,110 @@ import * as React from "react";
     title: "Perseus/Widgets/Radio/Option Status",
+    args: {
+        crossedOut: false,
+        checked: false,
+        correct: false,
+        previouslyAnswered: false,
+        reviewMode: true,
+        satStyling: false,
+    },
 }: Story);
 
-export const CheckedCorrectPreviouslyAnswered = (
-    args: StoryArgs,
-): React.Node => {
-    return (
-        <OptionStatus
-            crossedOut={false}
-            checked={true}
-            correct={true}
-            previouslyAnswered={true}
-            reviewMode={true}
-            satStyling={false}
-        />
-    );
-};
-
-export const CheckedCorrectNotPreviouslyAnswered = (
-    args: StoryArgs,
-): React.Node => {
-    return (
-        <OptionStatus
-            crossedOut={false}
-            checked={true}
-            correct={true}
-            previouslyAnswered={false}
-            reviewMode={true}
-            satStyling={false}
-        />
-    );
-};
-
-export const CheckedNotCorrectPreviouslyAnswered = (
-    args: StoryArgs,
-): React.Node => {
-    return (
-        <OptionStatus
-            crossedOut={false}
-            checked={true}
-            correct={false}
-            previouslyAnswered={true}
-            reviewMode={true}
-            satStyling={false}
-        />
-    );
-};
-
-export const CheckedNotCorrectNotPreviouslyAnswered = (
-    args: StoryArgs,
-): React.Node => {
-    return (
-        <OptionStatus
-            crossedOut={false}
-            checked={true}
-            correct={false}
-            previouslyAnswered={false}
-            reviewMode={true}
-            satStyling={false}
-        />
-    );
-};
-
-export const NotCheckedCorrectPreviouslyAnswered = (
-    args: StoryArgs,
-): React.Node => {
-    return (
-        <OptionStatus
-            crossedOut={false}
-            checked={false}
-            correct={true}
-            previouslyAnswered={true}
-            reviewMode={true}
-            satStyling={false}
-        />
-    );
-};
-
-export const NotCheckedCorrectNotPreviouslyAnswered = (
-    args: StoryArgs,
-): React.Node => {
-    return (
-        <OptionStatus
-            crossedOut={false}
-            checked={false}
-            correct={true}
-            previouslyAnswered={false}
-            reviewMode={true}
-            satStyling={false}
-        />
-    );
+export const Interactive = (args: StoryArgs): React.Node => {
+    return <OptionStatus {...args} />;
 };
 
-export const NotCheckedNotCorrectPreviouslyAnswered = (
-    args: StoryArgs,
-): React.Node => {
-    return (
-        <OptionStatus
-            crossedOut={false}
-            checked={false}
-            correct={false}
-            previouslyAnswered={true}
-            reviewMode={true}
-            satStyling={false}
-        />
-    );
-};
-
-export const NotCheckedNotCorrectNotPreviouslyAnswered = (
-    args: StoryArgs,
-): React.Node => {
+export const AllPossibleOutputs = (args: StoryArgs): React.Node => {
```

Comment:
I like these kinds of stories especially for components that have a lot of variations.  It's nice being able to see all the variations at once.  Also, if we ever decide to use Chromatic for image comparison test it'll only count as a single snapshot which helps us keep costs down.